rules/import_middleman.bzl

```diff
@@ -0,0 +1,227 @@
+load("@build_bazel_rules_apple//apple:providers.bzl", "AppleFrameworkImportInfo")
+load("@build_bazel_rules_apple//apple:providers.bzl", "AppleBundleInfo", "AppleSupportToolchainInfo")
+
+_Provider = provider(fields = {
```

Comment:
Nit: Should this name be more descriptive instead of `_Provider`?

Reply:
SGTM